frontend/src/pages/destructionlist/review/DestructionListReview.loader.ts

```diff
@@ -76,9 +79,9 @@ export const destructionListReviewLoader = loginRequired(
           reviewItemsPromise,
           reviewResponsePromise,
           listReviewers(),
-          listZaken({
-            ...objParams,
-            in_destruction_list: uuid,
+          listDestructionListItems(uuid, {
+            "item-order_review_ignored": String(true),
```

Comment:
This seems odd, 

- The serialization can be done in the API function right?
- If thats for some kind of reason not desirable, we can just pass "true" as string?